pkg/cmd/add/add_app.go

```diff
@@ -61,6 +60,9 @@ var (
 		# Add an app
 		jx add app jx-app-jacoco
 
+		# Add an app using a chart repository prefix from the version stream 'charts/repositories.yml' file
```

Comment:
Here things get confusing for me. So far this command is documented as being responsible for installing "apps" which are similar to "addons". So without any further context, I read this as "install an addon from a chart repository prefix from the version stream 'charts/repositories.yml' file". And not sure what the second part really means.

Do you have a plan how we get out of this naming mess?

Reply:
the plan is to always use app and remove addon: https://github.com/jenkins-x/enhancements/tree/master/proposals/1